python/build_wheel.sh.in

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
Consider using a Python script to replace this shell script in the future as that would be more portable across various systems. You could also use CMake file instead of Shell script like it's done in nGraph TensorFlow brdge: 
https://github.com/NervanaSystems/ngraph-tf/blob/master/python/CreatePipWhl.cmake